bundles/framework/divmanazer/component/SelectList.js

```diff
@@ -60,86 +60,103 @@ Oskari.clazz.define('Oskari.userinterface.component.SelectList',
       });
       return el;
     },
-    /**@method selectFirstValue
-    *  makes the second value show instead of placeholder text
+    /** @method selectFirstValue
+    *   Select the first non-placeholder value
     */
-    selectFirstValue: function(){
+    selectFirstValue: function () {
       var chosen = this.element.find('select');
       chosen.find('option:nth-child(2)').attr('selected', 'selected');
```

Comment:
The second one is always the first? So there's always an "empty option" at first index?